# Add easybuild locations of dependencies
# December 13th 2018 by B. Hajgato (Free University Brussels - VUB)
--- libxml2-2.9.8/python/setup.py.orig	2018-03-05 16:54:29.000000000 +0100
+++ libxml2-2.9.8/python/setup.py	2018-12-13 09:06:59.345196079 +0100
@@ -56,11 +56,8 @@
 # - iconv.h
 # - libxslt/xsltconfig.h
 includes_dir = [
-"/usr/include",
-"/usr/local/include",
-"/opt/include",
-os.path.join(ROOT,'include'),
-HOME
+os.path.join(os.getenv("EBROOTLIBXML2"),"include"),
+os.path.join(os.getenv("EBROOTLIBICONV"),"include"),
 ];
 
 xml_includes=""
